Python/Product/ExternalProfilerDriver/Program.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Utility.CommandLine;
+
+namespace Microsoft.PythonTools.Profiling.ExternalProfilerDriver {
+    class Program {
+
+        // [Argument('p', "path")]
+        private static bool ReportVTunePath { get; set; }
+
+        // [Argument('n', "dry-run")]
+        private static bool DryRunRequested { get; set; }
+
+        // [Operands]
+        private static string[] RestArgs { get; set; }
+
+        static void PrintUsage() {
+        }
+
+        static int Main(string[] args)
+        {
+#if false
+            try
+            {
+                Arguments.Populate();
+            } catch (ArgumentException aex) {
+                Console.WriteLine($"Incorrect form of arguments: {aex.Message}");
+                return 1;
+            } catch (Exception ex)
+            {
+                Console.WriteLine("Unidentified error condition");
+                return 1;
+            }
+#endif
+
+            if (true || ReportVTunePath)
+            {
+                try
+                {
+                    Console.WriteLine($"The path of VTune is: {VTuneInvoker.VTunePath()}");
+                    return 0;
+                } catch (VTuneNotInstalledException ex)
+                {
+                    Console.WriteLine($"VTune not found in expected path: {ex.Message}");
+                    return 1;
+                }
+            }
+
+            string vtuneExec = VTuneInvoker.VTunePath();
+
+            VTuneCollectHotspotsSpec spec = new VTuneCollectHotspotsSpec()
+            {
+                WorkloadSpec = String.Join(" ", RestArgs)
+            };
+            string vtuneCollectArgs = spec.FullCLI();
+
+            VTuneReportCallstacksSpec repspec = new VTuneReportCallstacksSpec();
+            string vtuneReportArgs = repspec.FullCLI();
+
+            VTuneCPUUtilizationSpec reptimespec = new VTuneCPUUtilizationSpec();
+            string vtuneReportTimeArgs = reptimespec.FullCLI();
+
+            if (!DryRunRequested)
+            {
+                ProcessAsyncRunner.RunWrapper(vtuneExec, vtuneCollectArgs);
+                ProcessAsyncRunner.RunWrapper(vtuneExec, vtuneReportArgs);
+                ProcessAsyncRunner.RunWrapper(vtuneExec, vtuneReportTimeArgs);
+            }
+
+            string tempOutDir = Environment.GetEnvironmentVariable("USERPROFILE");
```

Comment:
This is one of the worst places to write output to. If it's temporary, use `Path.GetTempPath()` (and probably create a randomly named subdirectory to avoid overwriting anything), or allow it to be overridden from the command line (which will make testing easier, as it can be redirected to a non-system wide location).